_inc/lib/tracks/class.tracks-client.php

```diff
@@ -158,7 +158,7 @@ static function get_anon_id() {
 				$anon_id = 'jetpack:' . base64_encode( $binary );
 
 				if ( ! headers_sent() ) {
-					setcookie( 'tk_ai', $anon_id );
+					setcookie( 'tk_ai', $anon_id, time() + 1000, '/' );
```

Comment:
Won't this mean that the cookie expires about 15 minutes after it's set? This seems like it would cause some disruption. Maybe we could check if the value is correct or not and use that to update the cookie, instead of just making the old one expire.

Reply:
How would it cause a disruption?